mono/metadata/object.c

```diff
@@ -4571,14 +4571,15 @@ mono_unhandled_exception (MonoObject *exc_raw)
 	HANDLE_FUNCTION_ENTER ();
 	MONO_HANDLE_DCL (MonoObject, exc);
 	error_init (&error);
-	mono_unhandled_exception_checked (exc, &error);
+	mono_unhandled_exception_checked (exc, TRUE, &error);
 	mono_error_assert_ok (&error);
 	HANDLE_FUNCTION_RETURN ();
 }
 
 /**
  * mono_unhandled_exception:
  * @exc: exception thrown
+ * @fatal: if true abort execution
```

Comment:
I think we changed to doxygen-style API docs (`\param fatal` etc) at some point (see `mono_unhandled_exception`, above)